### DISCUSSION TITLE
como subo todas las practicas de Abogabot

### DISCUSSION THREAD

**Discussion by yes05** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#190] [General]
Hola que tal como subo todos mis trabajos de Abogabot a github

**Comment by jRodPerez13** [comment]
De la misma forma que subiste tus katas a github, asi será siempre

**Reply by yes05** [reply]
osea solo los dejo ahi, no tengo que enviar un link de que ya lo subi ?

**Reply by BrunoRamirezG** [reply]
En Teams, subieron un archivo donde esta el formulario. :)

**Comment by AngelCruzO** [comment]
Pues los documentos los metes directo a tu repositorio, si son enlaces los puedes colocar en un README con hipervinculos.

**Reply by yes05** [reply]
muchas gracias, pero entonces ahi las dejo y ya verdad, ya las subi pero pues ellos van a ver despues mis trabajos? osea no mando el link de que ya los subi

**Reply by AngelCruzO** [reply]
Tienes que llenar el forms desde este [link](https://forms.office.com/r/aAdUCJrGL4) y ya con eso entregas tus ejercicios

**Reply by ArmandoRiveraT** [reply]
Tienes que mandar los links al siguiente formulario: https://forms.office.com/pages/responsepage.aspx?id=BUvlSn63JEKu8YZhQi4IFm-27paEuYNAjqZ0w0oHo0pUMk9TWUQ1MDAxT1k1MTMyN0VMNU45QjFFQi4u

**Reply by yes05** [reply]
MIL GRACIAS EN VERDAD MIL GRACIAS!!!!!!!!!!!!!!

**Comment by JuanSalvador73** [comment]
Simplemente las arrastras desde una ventana de tu computadora a tu repositorio creado, en donde dice "add files" en un dropdown menú.

**Comment by davidtoString** [comment]
Crea un repositorio aquí en Github 

Sube tus archivos ahí

Llena en formulario de entrega

**Reply by yes05** [reply]
Muchas gracias en verdad! Esa era mi duda
